camligorkem/assignment2/BinaryNode.java

```diff
@@ -0,0 +1,35 @@
+public class BinaryNode<T> {
+    private T data;
+    private BinaryNode<T> left;
+    private BinaryNode<T> right;
+
+    public BinaryNode(T data) {
+        this.data = data;
+        left = null;
```

Comment:
You don't need to set those to null, java does that by default :)

Reply:
done!